staging/src/k8s.io/client-go/tools/pager/pager_test.go

```diff
@@ -301,14 +301,14 @@ func TestListPager_EachListItem(t *testing.T) {
 		{
 			name:                "cancel context while processing",
 			fields:              fields{PageSize: 10, PageFn: (&testPager{t: t, expectPage: 10, remaining: 51, rv: "rv:20"}).PagedList},
-			want:                list(3, "rv:20"), // all the items <= the one the processor returned an error on should have been visited
+			want:                list(10, "rv:20"), // all the items <= the one the processor returned an error on should have been visited
 			wantErr:             true,
 			cancelContextOnItem: 3,
 		},
 		{
 			name:      "panic processing item",
 			fields:    fields{PageSize: 10, PageFn: (&testPager{t: t, expectPage: 10, remaining: 51, rv: "rv:20"}).PagedList},
-			want:      list(3, "rv:20"), // all the items <= the one the processor returned an error on should have been visited
+			want:      list(51, "rv:20"), // all the items <= the one the processor returned an error on should have been visited
 			wantPanic: true,
 		},
 	}
```

Comment:
This test is just plain wrong. It should have had `processorPanicOnItem: 3` set. It overlaps with the above `error processing item` and so should just be removed. My apologies.